src/modules/editTask.js

```diff
@@ -0,0 +1,15 @@
+const editTask = (tasks) => {
+  for (let i = 0; i < tasks.taskArray.length; i += 1) {
+    const taskName = document.querySelectorAll('.task-name');
+    taskName[i].addEventListener('keydown', (e) => {
+      if (e.keyCode === 13) {
+        e.preventDefault();
+      }
+    });
+    taskName[i].addEventListener('input', () => {
+      tasks.editTask(i, taskName[i].innerHTML);
+    });
+  }
```

Comment:
- Hey, let's try to make the edit function work properly by making it to be updated on the localStorage to improve the functionality.